src/web/mjs/engine/Manga.mjs

```diff
@@ -143,8 +143,9 @@ export default class Manga extends EventTarget {
             .then( chapters => {
                 let chapterFormat = Engine.Settings.chapterTitleFormat.value;
                 // de-serialize chapters into objects
+                let formatRegex = this.connector._getFormatRegex();
```

Comment:
May skip local variable, as this is only used once

Reply:
it's used once for every chapter
but ok